server/models/articles.js

```diff
@@ -0,0 +1,57 @@
+const getArticleModel = (sequelize, DataTypes) => {
+  const flags = {
+    freezeTableName: true
+  };
+  const articleSchema = {
+    id: {
+      type: DataTypes.UUID,
+      defaultValue: DataTypes.UUIDV4,
+      primaryKey: true,
+      unique: true,
+      allowNull: false
+    },
+    title: {
+      type: DataTypes.TEXT,
+      allowNull: false
+    },
+    authorId: {
+      type: DataTypes.UUID,
+      allowNull: false
+    },
+    categoryId: {
+      type: DataTypes.UUID,
+      allowNull: false
+    },
+    body: {
+      type: DataTypes.TEXT,
+      allowNull: false
+    },
```

Comment:
If this is meant for the article content, why not name it `content` instead of `body`?